region/maridia/outer/Mama Turtle Room.json

```
@@ -561,6 +561,39 @@
         "Samus will sometimes be forced onto Mama Turtle's back, which can force a third hit; this can be prevented by holding forward as she hits Samus."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "Gravity",
+        "h_CrystalFlashForReserveEnergy",
+        {"canShineCharge": {"usedTiles": 19, "gentleUpTiles": 2, "openEnd": 1}},
```

could add 19 with the door open
```suggestion
        {"canShineCharge": {"usedTiles": 18, "gentleUpTiles": 2, "openEnd": 1}},
```